tuist/tuist
# Support unit test depending on an app extension

Resolves <https://github.com/tuist/tuist/issues/6553>

### Short description 📝

Unit test target for app extension was not supported although it was a valid combination.

### How to test the changes locally 🧐
Run the acceptance test `GenerateAcceptanceTestiOSAppWithExtensionAndTests` and po the generated xocde project workspace and open it. Select the iOS target you will see the test target. You can also run the test.


### Contributor checklist ✅


### Reviewer checklist ✅
